Kritika_Sachdeva/Calculator/styles.css

```diff
@@ -0,0 +1,122 @@
+body{
+    background-image: linear-gradient(to right, black, lightblue);
+    margin: 0;
+
+}
+
+.container{
+    background-color: rgba(0, 0, 0, 0.842);
+    height: 450px;
```

Comment:
is this responsive design?

Reply:
No, it wasn't.
I've made the required changes.